src/lib/y2journal/query_presenter.rb

```diff
@@ -106,11 +111,11 @@ def self.intervals
 
       intervals << { value: Hash, label: _("Between these dates") }
 
-      label = _("Since system's boot (%s)") % boots.last[:timestamps]
+      label = _("Since system's boot (%s)") % localize_timestamp(boots.last[:timestamps])
       intervals << { value: "0", label: label }
 
       if boots.size > 1
-        label = _("From previous boot (%s)") % boots[-2][:timestamps]
+        label = _("From previous boot (%s)") % localize_timestamp(boots[-2][:timestamps])
```

Comment:
Hmmm, as far as I can see, `boots.last[:timestamps]` returns a string with two timestamps separated by a given separator.

In my opinion, the presenters should not deal with that kind of details of the journalctl output. It's clearly out of its responsibilities.

The class `Y2Journal::Query` is the one that calls the journalctl command and should be the responsible of parsing that output and creating a couple of `Time` objects out of it.

Then, this presenter only have to call `Yast::Builtins.strftime` for each one of those `Time` objects.

Reply:
Hello @ancorgs! I realized only today that I had not replied to your review comment after pushing a new commit back in December. Could you please review the code again and suggest any further changes needed?